packages/entity-list/src/components/PaginationPanel/PaginationPanel.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Button, ButtonGroup} from 'tocco-ui'
+
+import {StyledPaginationPanel} from './StyledPaginationPanel'
+
+const PaginationPanel = props => {
+  const {
+    changePage,
+    components,
+    totalPages,
+    currPage
+  } = props
+
+  return (
+    <StyledPaginationPanel>
+      {components.totalText}
```

Comment:
`components` is not declared as a required prop and thus accessing `totalText`could lead to a null pointer exception.

the same applies to the other props.